changelogs/fragments/9845-consul_token_idempotency.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - consul_token - Fix idempotency when ``policies`` or ``roles`` are supplied by name (https://github.com/ansible-collections/community.general/issues/9841 https://github.com/ansible-collections/community.general/pull/9845).
```

Comment:
```suggestion
  - consul_token - fix idempotency when ``policies`` or ``roles`` are supplied by name (https://github.com/ansible-collections/community.general/issues/9841, https://github.com/ansible-collections/community.general/pull/9845).
```

Reply:
*sigh*, one day I will manage to get a PR in without you requesting changes. Not sure which year though.